### DISCUSSION TITLE
ESP8266 does not boot

### DISCUSSION THREAD

**Discussion by uraich** [micropython/micropython#12506] [ESP8266]
This is what I see:
load 0x40100000, len 30720, room 16 
tail 0
chksum 0xba
load 0x3ffe8000, len 996, room 8 
tail 12
chksum 0x2c
ho 0 tail 12 room 4
load 0x3ffe83f0, len 1080, room 12 
tail 12
chksum 0x43
csum 0x43
rf_cal[0] !=0x05,is 0xFF

 ets Jan  8 2013,rst cause:2, boot mode:(3,6)

Flashing seems to work without problems. I tried with firmware I compiled myself but also with ESP8266_GENERIC-20230426-v1.20.0.bin. Always the same story.
Any idea what is going wrong? I did run micropython on the board before (long time ago)

**Comment by glenn20** [comment]
Just something to try... flash the GENERIC_1M firmware and see if that works.

**Comment by bidrohini** [comment]
You can see the common troubleshooting guideline and a thread from the old micropython forum. I am sharing the links. You can check and see if they help.
https://randomnerdtutorials.com/esp8266-troubleshooting-guide/
https://forum.micropython.org/viewtopic.php?t=12869

By the way, if you ever consider making  a self balancing robot using ESP32, you can follow this design.

https://www.pcbway.com/project/shareproject/Self_Balancing_Robot_Using_ESP32_1171d9cb.html

**Comment by uraich** [comment]
Finally I managed to flash ESP8266_GENERIC-20230426-v1.20.0.bin and that one works after having erase the ESP8266 flash.
I have no clue however, what is going wrong when I compile the firmware myself.
I am strictly following the instructions in the 8266 port README. 
It is not a big problem for me as I mainly use ESP32 chips for which everything works fine.
